docs/src/config-file.md

```diff
@@ -181,6 +181,12 @@ postgres:
 
 # Publish PMTiles files from local disk or proxy to a web server
 pmtiles:
+  # allows forcing path style URLs for S3 buckets [default: false]
+  #
+  # A path style URL is a URL that uses the bucket name as part of the path (`mys3.com/somebucket`) instead of the hostname (`somebucket.mys3.com`).
+  force_path_style: false
+  # To send requests anonymously for publicly available buckets [default: false]
+  no_credentials: false
```

Comment:
i keep tripping over this name - `no_credentials: false` is a double negative - can we perhaps rename it to something more readable, e.g. `allow_anonymous` or `require_credentials` or ... ?

Reply:
`require_credentials: true` (default: `true`) is closest to what this does.
Will change this accordingly.